tests/unit/field/test_vector3d.cxx

```diff
@@ -163,8 +166,12 @@ TEST_F(Vector3DTest, SetLocationZLOW) {
 }
 
 TEST_F(Vector3DTest, SetLocationVSHIFT) {
-  Vector3D vector;
-  vector.x.getMesh()->StaggerGrids = true;
+  FakeMesh local_mesh{Vector3DTest::nx,Vector3DTest::ny,Vector3DTest::nz};
+  local_mesh.StaggerGrids = true;
+  local_mesh.setCoordinates(nullptr, CELL_XLOW);
+  local_mesh.setCoordinates(nullptr, CELL_YLOW);
+  local_mesh.setCoordinates(nullptr, CELL_ZLOW);
+  Vector3D vector(&local_mesh);
   EXPECT_EQ(vector.getLocation(), CELL_CENTRE);
   EXPECT_NO_THROW(vector.setLocation(CELL_VSHIFT));
```

Comment:
Note `FakeMesh` has `xstart=1` so if we tried to let the actual coordinates be created here we'd find an assert fails. So whilst this (setting nullptr coordinates) does allow us to proceed with the test in some sense it's hiding a potential issue -- for the unit tests here it's probably OK but is something we might need to address for other interpolation tests.